src/Compilers/VisualBasic/Portable/Compilation/VisualBasicCompilation.vb

```diff
@@ -2716,6 +2716,13 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End Select
         End Function
 
+        Public Overrides Function GetRequiredLanguageVersion(diagnostic As Diagnostic) As String
+            ' The driving scenario for this API (upgrading project based on language versions reported in diagnostics)
```

Comment:
If throwing broke many IDE tests, I'd then ask the question of why we were calling an API we knew wouldn't do anything in the first place. :smile: